defaults/main.yml

```diff
@@ -3,15 +3,19 @@
 
 # We'll end up fetching
 # {{ ojs_download_url }}{{ ojs_tarball }}
-ojs_version: "3.1.0-1"
-ojs_md5sum: "fcb665e19ae25c204c3ddc7efa86d232"
+sclphp_version: "rh-php72"
```

Comment:
I'd suggest letting the php role own these variables, and possible adding a minimum version check on the installed PH. It seems like this is intended to suggest a minimum version, but nothing really enforces that.